README.md

```diff
@@ -683,7 +683,32 @@ LibV8 itself is already [beyond version 7](https://github.com/cowboyd/libv8/rele
 ### HMR
 Hot Module Replacement is [possible with this gem](https://stackoverflow.com/a/54846330/193785) as it does just pass through to Webpacker. Please open an issue to let us know tips and tricks for it to add to the wiki.
 
-Sample repo that shows HMR working with `react-rails`: [https://github.com/edelgado/react-rails-hmr](https://github.com/edelgado/react-rails-hmr)
+Similar to `useContext`, you can pass the webpack context into `useHotReload` to enable hot reload:
+1. install [react-hot-loader](https://github.com/gaearon/react-hot-loader) and [@hot-loader/react-dom](https://github.com/hot-loader/react-dom)
+2. add the following to your webpack config in dev:
+```js
+{
+  module: {
+    rules: [
+      {
+        test: /\.(jsx|tsx)?$/,
+        use: ["react-hot-loader/webpack"],
+      },
+    ],
+  },
+  resolve: {
+    alias: {
+      "react-dom": "@hot-loader/react-dom",
+    },
+  },
+}
+```
+3. in your entry file, usually where you call `ReactRailsUJS.useContext` already, call `useHotReload`:
+```js
+var ReactRailsUJS = require("react_ujs")
+var myCustomContext = require.context("custom_components", true)
+ReactRailsUJS.useHotReload(myCustomContext)
+```
```

Comment:
Let me know if this is concise and simple to follow. Happ to pull into a separate file if requested